lib/user-repo.js

```diff
@@ -4,22 +4,24 @@ const { GH_GENERAL } = require('@gh-conf/gh-conf-constants');
 
 const ParseConfig = require('./parse-config');
 
+const getCurrRepo = () => {
```

Comment:
Can you revert these changes?
We do not want to merge two PR in one.
Just have the removing yarn.lock file in this PR

Reply:
ok, sorry